website/docs/usage/visualizers.md

```diff
@@ -389,6 +392,16 @@ in displaCy's format as a dictionary (instead of `Doc` objects).
 }
 ```
 
+```python
+### SPANS input with knowledge base links
+{
+    "text": "But Google is starting from behind.",
+    "spans": [{"start": 4, "end": 10, "start_token": 0, "end_token": 1, "label": "ORG", "kb_id": "Q95", "kb_url": "https://www.wikidata.org/entity/Q95"}],
```

Comment:
Does it need the `start_token` and `end_token`? I guess I'm mainly asking whether the differences between the `ENT input` (above) and this `SPANS input` are due to fundamental differences in the spans/ents functionality?

Also, I wonder whether we should have the same example as for ents, or perhaps have an example with overlapping spans to highlight why `spans` would be useful.

Reply:
Let me look into it more. I basically took the output of `displacy.parse_spans` and went with that, but I'm not sure how those are used in the actual render function.

Also, agreed on the example, I'll try and come up with one that has overlaps.

Reply:
Well, it's much easier to answer these questions after I worked on https://github.com/explosion/spaCy/pull/11092 + https://github.com/explosion/spaCy/pull/10994. 😄 

We only use `start_token` and `end_token`. However, there's an inconsistency in how we refer to the token start/end between NER and Spans. If we wanted to be consistent, I can think of a few options: 
1. Rename `start` and `end` in the displaCy NER parser /  data structure to be `start_token` and `end_token`.
2. Remove `start` and `end` from the span data structure, then rename current (`start_token`, `end_token`) to (`start`, `end`) for consistency with NER. If we ever need the character-level information from a span, this could create an issue.

Reply:
We should update the example here to contain only the information needed, which will at least highlight the difference/inconsistency. And it would be nice to have an example with overlapping spans to better illustrate the usefulness of this option.

If we'd want to revise the actual underlying data structures, it's probably best/safest to defer that to v4 - let me add a note to our internal backlog for that.

Reply:
Sounds good, I'll update to the [example we have in the visualizers documentation](https://spacy.io/usage/visualizers#span), because it's nice and short.